state/account.go

```diff
@@ -118,9 +205,11 @@ func (st *Account) Clone() *Account {
 	s := *st
 	s.Balance = nil
 	s.Balance = new(big.Int).Set(st.Balance)
-	s.VotingWeight = nil
-	if st.VotingWeight != nil {
-		s.VotingWeight = new(big.Int).Set(st.VotingWeight)
+	s.accountType = st.accountType
+	s.nonce = st.nonce
```

Comment:
don't need these 2 lines since `s := *st`

Reply:
same here, l219,221,222 could be removed